### PR TITLE
Adds v1.13.0 Release Blog Post

### PR DESCRIPTION
**Pull request by danehans**
Adds a blog post for the v1.13.0 release.

### CODE REVIEW

**Review by skriss — APPROVED** [review]

File: site/_posts/2021-02-25-contour_v1130.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+---
+title: Announcing Contour v1.13.0
+excerpt: Contour 1.13 adds Gateway API and global rate limiting support.
+author_name: Daneyon Hansen
+author_avatar: /img/contributors/daneyon-hansen.png
```
Comment (jonasrosland) [inline site/_posts/2021-02-25-contour_v1130.md:5]: This should be /img/contributors/daneyon-hansen.jpg

**Review by jonasrosland — CHANGES_REQUESTED** [review]
Just one tiny change, other than that this looks great, thank you @danehans !

**Comment by danehans** [comment]
@jonasrosland commit `73a997f` resolved https://github.com/projectcontour/contour/pull/3413#discussion_r583207766.

**Review by jonasrosland — APPROVED** [review]
LGTM, approved!

**Review by youngnick — APPROVED** [review]

**Review by sunjayBhatia — APPROVED** [review]

**Pull request merged**